apps/bazel-is-cool/BUILD.bazel

```diff
@@ -0,0 +1,67 @@
+load("@npm//@angular/cli:index.bzl", "ng", "ng_test")
+
+filegroup(
+    name = "srcs",
+    srcs = glob([
+        "src/*.ts",
+        "src/*.css",
```

Comment:
Maybe just glob everything under `srcs`? Or at least under `srcs/assets`. We need to take into account all valid src files such as `json` `scss` `less` `js` `json` ect. So we can either account for all file extensions supported by our builder of choice or just say you get everything under `src`